Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/ReportCategory.cs

```diff
@@ -263,6 +253,11 @@ public static class ReportCategory
 
         public const string AUDIO = nameof(AUDIO);
 
+        /// <summary>
+        ///     Generic UI category
+        /// </summary>
+        public const string UI = nameof(UI);
```

Comment:
Is this category for our UIs or for the scenes UI? Remember that we already has a category called `SCENE_UI`.

Reply:
No, it's a category for Unity UI